client/src/Footer.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Container, Header, List, Grid, Segment } from "semantic-ui-react";
+
+export const Footer = () => (
+  <Segment inverted vertical style={{ padding: "1em 0em", marginTop: "6em" }}>
+    <Container>
+      <Grid divided inverted stackable>
+        <Grid.Row>
+          <Grid.Column width={4}>
+            <Header inverted as="h4" content="Team" />
+            <List link inverted>
+              <List.Item as="a">Adnan Oglah</List.Item>
+              <List.Item as="a">Blaise Lubuma</List.Item>
+              <List.Item as="a">Mariama Sillah</List.Item>
+              <List.Item as="a">Mohammed Lafiteh</List.Item>
+              <List.Item as="a">Victoria Kosenko</List.Item>
+            </List>
+          </Grid.Column>
+          <Grid.Column width={7}>
+            <Header as="h4" inverted>
+              Broca Translation
+            </Header>
+            <p>
+              Broca Translation is an open source project hosted on{" "}
+              <a href="https://github.com/CodeYourFuture/scot-broca-translators">
+                Github.
```

Comment:
Can you add the Github icon from Semantic UI before the text Github? Also, can you make the link white and underline instead of blue?

Reply:
it is done

Reply:
Can you put the icon of GitHub  right before the text Github instead?

Reply:
I added it before, but the underline will be broken between the icon and the text Github:(, any way to fix it?

Reply:
You can put the icon before the link, so it's not underline (before the  <a> tag)

Reply:
Great , thank you Loic, it is done now.